### DISCUSSION TITLE
Can you disable hotkeys on Firefox?

### DISCUSSION THREAD

**Discussion by DavisStanko** [darkreader/darkreader#8201] [Q&A] [answered]
I recently switched over from Chrome to Firefox. On Chrome, I had every single hotkey disabled except add site to inverted list. On Firefox I can't seem to find a way to remove hotkeys for things such as toggle extension. This leads to conflicting key binds with other applications. Am I missing something?

**Comment by Gusted** [comment, marked as answer]
See: https://support.mozilla.org/en-US/kb/manage-extension-shortcuts-firefox

**Reply by DavisStanko** [reply]
Thanks a lot! This worked.
